modules/core/src/main/java/org/apache/ignite/lang/ErrorGroups.java

```diff
@@ -400,4 +400,15 @@ public static class Compute {
         /** Class loader error. */
         public static final int CLASS_LOADER_ERR = COMPUTE_ERR_GROUP.registerErrorCode((short) 2);
     }
+
+    /**
+     * File transfer error group.
+     */
+    public static class FileTransfer {
+        /** File transfer error group. */
+        public static final ErrorGroup FILE_TRANSFER_ERR_GROUP = ErrorGroup.newGroup("FILE_TRANSFER", (short) 17);
+
+        /** Generic error. */
+        public static final int GENERIC_ERR = FILE_TRANSFER_ERR_GROUP.registerErrorCode((short) 1);
```

Comment:
Please, avoid using generic error codes, just because it does not make much sense to the end user. This must be used with great care.